interfaces/FrontMatter.ts

```diff
@@ -12,6 +12,10 @@ export default interface FrontMatter {
     tags: string[]
     author?: string | null
 
+    /** First publication date and updated publication date */
+    publicationDate?: string | null 
```

Comment:
Should we make this required? I guess it depends on how we feel about having a timestamp on everything — to me, it makes sense to require it, but if you envision a case where we wouldn't include it, making it optional seems like the way to go.

Reply:
I think it should be required but I was having trouble with requiring it to be required

Reply:
Yeah, that makes sense! Let's leave it for now then; we can always change it later, if we decide it really should be required.